code/datums/bodypart_overlays/bodypart_overlay.dm

```diff
@@ -40,22 +40,22 @@
 */
 /datum/bodypart_overlay/proc/mutant_bodyparts_layertext(layer)
 	switch(layer)
-		if(BODY_BEHIND_LAYER)
+		if(-BODY_BEHIND_LAYER)
 			return "BEHIND"
-		if(BODY_ADJ_LAYER)
+		if(-BODY_ADJ_LAYER)
 			return "ADJ"
-		if(BODY_FRONT_LAYER)
+		if(-BODY_FRONT_LAYER)
 			return "FRONT"
 
 ///Converts a bitflag to the right layer. I'd love to make this a static index list, but byond made an attempt on my life when i did
 /datum/bodypart_overlay/proc/bitflag_to_layer(layer)
 	switch(layer)
 		if(EXTERNAL_BEHIND)
-			return BODY_BEHIND_LAYER
+			return -BODY_BEHIND_LAYER
 		if(EXTERNAL_ADJACENT)
-			return BODY_ADJ_LAYER
+			return -BODY_ADJ_LAYER
 		if(EXTERNAL_FRONT)
-			return BODY_FRONT_LAYER
+			return -BODY_FRONT_LAYER
```

Comment:
Mob layers are inverted before being drawn, which I inverted for mutant overlays but not the simple ones. I changed it here so it can't happen again (and sixpacks wont draw over clothing, even though it looked extremely metal)